### ISSUE TITLE
[Feature] Enhanced geospatial support

### ISSUE THREAD

**Issue by austinmw** [open-mmlab/mmdetection#9335]
### What's the feature?

Hi, I recently tried out the library [TorchGeo](https://github.com/microsoft/torchgeo) and found it very useful for geospatial vision models, particularly because of its support for many datasets (through support for [Radiant MLHub](https://mlhub.earth/)), and the geospatial samplers and transforms implemented.

However MMDetection (and MMRotate, MMSegmentation), offer a much larger choice for models.

I'm curious if there's a way to be able to take advantage of the useful geospatial datasets and samplers of TorchGeo, while training a model in OpenMMLab?

Are there any ways that geospatial support could be enhanced similar to features available in TorchGeo, or maybe instead a way to train on top of TorchGeo dataloaders? I see that Detectron2 support is being added, so I figured this library might be flexible enough for something like this. 

Thanks for reading!

**Comment by BIGWangYuDong** [comment]
Thank you very much for your suggestion, if you have time it is very hopeful that you can contribute algorithms to MMDetection!
